### DISCUSSION TITLE
Plans to integrate with intellij or vscode?

### DISCUSSION THREAD

**Discussion by JoelCourtney** [helix-editor/helix#8466] [Ideas] [answered]
I loved helix when I tried it, but the switch wasn't worth it because I spend most of my time in IntelliJ with vim emulation; so I wouldn't have enough time to internalize helix's keybindings. Are there any plans to make an Intellij or vscode plugin? I would absolutely swoon.

**Comment by David-Else** [comment, marked as answer]
There is a vscode Helix keybindings plugin in development: https://marketplace.visualstudio.com/items?itemName=silverquark.dancehelix

Not sure if it is time for you to swoon yet, but maybe a slight tremor of excitement :)

**Reply by lewiszlw** [reply]
Do you know any plugin for intellij?

**Comment by tastyherring** [comment]
I like helix, but I can't see how I can switch either until there is an intellij plugin.

**Comment by g5becks** [comment]
This reddit thread might be of interest to you guys. Seems like this is possible via the IdeaVim plugin

https://www.reddit.com/r/HelixEditor/comments/14aqztf/helix_in_intellij/

**Comment by mark-at-sc** [comment]
Seems as though the original vscode Helix keybinding plugin is abandoned (no updates since 11/2022)?
https://marketplace.visualstudio.com/items?itemName=silverquark.dancehelix

I saw this one which is active and seems to have gained traction:
https://marketplace.visualstudio.com/items?itemName=jasew.vscode-helix-emulation
